docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md

```diff
@@ -295,8 +302,6 @@ Although the openstack-cloud-controller-manager was initially implemented with N
 
 NOTE:
 
-* When using `ovn` provider service has limited scope - `create_monitor` is not supported and only supported `lb-method` is `SOURCE_IP`.
```

Comment:
Can you keep the note extended with the information that OVN Octavia Provider supports health monitors only starting from Wallaby [1]?

[1] https://opendev.org/openstack/ovn-octavia-provider/commit/8bbd8f1b62d7811433079bff8ce940fbdd041a0c

Reply:
Keep this node or add a note to the `loadbalancer.openstack.org/enable-health-monitor` annotation doc (or both)? I would add it (only) to the annotation to keep them close together.

Reply:
Sure, we can put it at the annotation. Thanks!